src/webrtc/call.ts

```diff
@@ -676,6 +676,9 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
     private async initOpponentCrypto(): Promise<void> {
         if (!this.opponentDeviceId) return;
         if (!this.client.getUseE2eForGroupCall()) return;
+        // We (currently) don't speak e2e with foci. It's debateable whether there would be
+        // any benefit in doing so.
+        if (this.isFocus) return;
```

Comment:
Just to be sure. We explicitly do not want encrypted messages with the SFU?

Reply:
Not currently - we do the same check later to see if we're talking to a focus and send unencrypted messages.